test/test_site/expected/siteData.json

```diff
@@ -0,0 +1,16 @@
+{
+  "pages": [
+    {
+      "title": "L - Hello World",
```

Comment:
Where's `L` from?

Reply:
Was testing prefixes and forgot to revert.

Reply:
Looks like we should add a check for **siteData.json** in [test.js](https://github.com/MarkBind/markbind/blob/8bf51b9b9b687f2c29b0d774ea2fe5410240a130/test/test_site/testUtil/test.js#L22).

Reply:
Will add this